ref/requirements.md

```diff
@@ -81,9 +77,7 @@ Anbox Cloud currently supports the following LXD versions:
 <a name="juju-version"></a>
 ### Juju version
 
-Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**.
-
-[note type="information" status="Note"]If you are using Anbox Cloud 1.11 or earlier, the required Juju version is **2.8**.[/note]
+Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**. If you are using Anbox Cloud 1.11 or earlier, the required Juju version is **2.8**.
```

Comment:
Shouldn't we remove old versions info such as this one?

Reply:
@adglkh Is it okay to remove this? I remember discussing something about this in Prague but am not sure whether we concluded on that. Could we still have users using versions earlier than 1.13?

Reply:
Maybe ERT and a few other users were still using 1.13 .@morphis could double-check on this too.
I don't think people could get Anbox Cloud 1.11 running on AWS due to a couple of changes in kernel and lxd. 
However to me, for versioning and history-tracking purposes, I'd suggest keeping them as they are in the doc.